fuseimg/utils/visualization/tests/inputs/detection/instances_val2014_fakesegm100_results.json

```diff
@@ -0,0 +1 @@
nts":"WUd6`0_>1K50000005Kjad2"},"score":0.863},{"image_id":1270,"category_id":44,"segmentation":{"size":[480,640],"counts":"Yn_6;d>5FBgAa0Y>300000^OgA=_>O1BTkg2"},"score":0.431},{"image_id":1270,"category_id":1,"segmentation":{"size":[480,640],"counts":"Vlf03S>6ZBNe=8SBKk=f0N2L4M4L3M3M3L4M3M3L4YLZNQJi1b5hNVJ\\1_5PO]JS1c5mNYJV1h5kNPITOVOT2j7lN\\HBEd1o7QOPHDL\\1U8j0hGkNGQNc8V3aGjNl8W1QGjNP9a31M4M2N3K4L5K4L5K4M4OVOZKZGg4f8^KUGa4l8dKoF\\4R9hKiFX4W9mKdFR4]9h0001N1L5J6K51O2M3N2N2N2N2O1N2N2N2O1N2N4L5L5J>B8H6K4K8HbI^H`5a7]JbHV54RJ[7e0cHo4U8kJoG^1Ja1Z8mLoG_1M`1U8nLPHc1M[1V8nLoGh1LW1Y8lLnGm1KS1o8nNRGo0o8VNgF@<V2o8TNlFE7T2m8oMUGM3m1i8oM\\G47_1_8TNcG<j0c0d7jNjGe0T1Kd86aGC`8?P311O4K47FN1O2N1O2N1O2N3L4MchW7"},"score":0.724},{"image_id":1270,"category_id":1,"segmentation":{"size":[480,640],"counts":"\\QZ35k>6SNFhD<U;EkD<S;EmD=P;DPE=n:DRE>j:DVE=i:CWE?g:AYE`0f:@ZEa0e:_O[Ea0d:@\\Ea0c:_O]Eb0b:^O^Ec0a:]O_Ee0_:\\O`Ef0^:ZObEl0Y:SOgET1R:lNnEU1R:jNnEW1R:hNnEY1R:fNmE\\1S:cNmE]1T:bNlE^1V:`NQEZ2[;?1O001O1O1N2N2N2M3N3L3O2M2O2M4M3kNlB7[>DQZ]5"},"score":0.802},{"image_id":1270,"category_id":15,"segmentation":{"size":[480,640],"counts":"Q76j>00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000oh]8"},"score":0.13},{"image_id":1270,"category_id":15,"segmentation":{"size":[480,640],"counts":"PaT33m>8H6J0000000000000000000000000000O1000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000O10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000O10000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000O1000000000P`d3"},"score":0.634},{"image_id":1270,"category_id":1,"segmentation":{"size":[480,640],"counts":"_in23k>3lNLYC6e<KZC7e<IYC9g<H`BN3O6=U=0_BF<;T=4mBLR=4nBMP=4oBMDCX=`0RC7j<JTC;i<ESC`0k<l0O100T1nNM1OO2O0O1N2N4M6K5J2O1O2N1O2N1OPVn5"},"score":0.994},{"image_id":1270,"category_id":1,"segmentation":{"size":[480,640],"counts":"aZj4>b>7H6K4L2M4M2M2O00100?@`0A`a[4"},"score":0.563},{"image_id":1270,"category_id":1,"segmentation":{"size":[480,640],"counts":"cch35k>5K0O1000O1000ROJRC6m<8[BB47a=>^BBa=o0001O001O00001O001O0010O0001O001L4H>G9I7Hb\\T5"},"score":0.179},{"image_id":1270,"category_id":1,"segmentation":{"size":[480,640],"counts":"eQP16g>5L4M2OO1OHDnA:S>HlA7T>JlA5T>LlA3T>NlA2T>NlA1T>0lAOT>2lAMT>4lALT>4lAKU>5kAJU>7kAIU>7kAIg=3YB50He=8XB03Gd=j0\\BVOd=j0\\BVOc=k0]BTOd=l0\\BROe=?f0I7N20000O10000O2O000O2O1Ocfi7"},"score":0.713},{"image_id":1290,"category_id":1,"segmentation":{"size":[540,640],"counts":"j^n42h`02O1N2O1N3N1N2O1N2O1N2O2M2O1N2N2O1N2O2M2O1N2O1N2O1N3N1N2N2O1N2O1N2O2M2O1N2O1N2O1N3M2O1N2O1N2O1N3M2L4L4K5L4K5L4K6K4L4K5L4K5L4L5J5L4K5L4K5L4L5J5L4K5L4K5L4L4K6K4K5L4L4K5L4K6K4K5L4L21O2N2O1N2N2N2O1N2N2N2N2O1N2N2N2O1N2N2N2O1N2N2N2N2O1N2N2N2O1N2N2N2O1O1N2O1O1O1N2O1O1N2O1O1N2O1O1N2O1O1O0O2O1O1N2O1O1N2O1O1O1N2O1O1N2O1O1N2O1O1N2O1O1O1N2O1O1N2O1O1N2O1O1N2K5I7I7J6I7I7I7G9E;E;E;E;H8K5K5
```

Comment:
Can you also put a simple example also in fuse/data/README.md
And a link to the notebook below?